central/telemetry/billingmetrics/scheduler.go

```diff
@@ -0,0 +1,58 @@
+package billingmetrics
+
+import (
+	"time"
+
+	"github.com/stackrox/rox/central/sensor/service/pipeline/clustermetrics"
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/logging"
+)
+
+const period = 1 * time.Hour
+
+var (
+	log  = logging.LoggerForModule()
+	stop = concurrency.NewSignal()
+)
+
+func gather() {
+	ids, err := getClusterIDs()
+	if err != nil {
+		log.Debug("Failed to get cluster IDs for billing metrics snapshot: ", err)
+		return
+	}
+	log.Debugf("Cutting billing metrics for %d clusters: %v", len(ids), ids)
+	newMetrics := clustermetrics.CutMetrics(ids)
+	// Store the average values to smooth short (< 2 periods) peaks and drops.
+	if err := checkIn(average(previousMetrics, newMetrics)); err != nil {
```

Comment:
Looking at this now and given that we collected max values inside `period` and will be max-aggregating later over `period`s, I'm not sure we should do some random smoothening here. Let's drop it?